### DISCUSSION TITLE
Fonts.iu.edu is incredibly slow please upgrade to using a CDN

### DISCUSSION THREAD

**Discussion by Pinpaho** [indiana-university/rivet-source#452] [Questions] [answered]
## Describe the bug
fonts.iu.edu hurts SEO scores and site speed because it's not a CDN.

## Expected behavior
Font files should load in quickly.

## Relevant context (required)
Currently in the CSS everything pulls benton-sans from fonts.iu.edu which is just a host NOT a CDN. Please update rivet css font-face declarations to use a CDN.

## Steps to reproduce
none - this is a request

**Comment by levimcg** [comment, marked as answer]
@UIDvlpr Thanks for raising this issue. Unfortunately, due to the way we license fonts, they have to be served they have to be served from an iu.edu domain. So your meaning of "CDN" here is unclear to me. Do you mean some third party CDN that offers font hosting services, or optimizing the host we are using to server static assets like font files?

I'm happy to relay information back to the team that manages fonts.iu.edu. In the meantime, I"m going to convert this to an entry in the [Discussions area](https://github.com/indiana-university/rivet-source/discussions) of this repo as this isn't a bug or anything we can really fix right now.
